### PR TITLE
Feature/blogposts search

### PR DESCRIPTION
**Pull request by AndreaPiovanelli**
As described in #8569

### CODE REVIEW

File: src/Orchard.Web/Modules/Orchard.Blogs/Views/BlobPosts_NavigationTabs.cshtml [hunk 1]
```diff
@@ -0,0 +1 @@
+﻿aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa
```
Comment (sebastienros) [inline src/Orchard.Web/Modules/Orchard.Blogs/Views/BlobPosts_NavigationTabs.cshtml]: left-over

**Comment by AndreaPiovanelli** [comment]
@sebastienros I removed the leftover you pointed out and answered your questions on issue #8569; should we merge this pr?

**Comment by sebastienros** [comment]
Can't the feature be part of the Blogs module instead of the Search one? Just a matter of what depends on what.

**Comment by AndreaPiovanelli** [comment]
> Can't the feature be part of the Blogs module instead of the Search one? Just a matter of what depends on what.

We chose to add the feature to Orchard.Search module because there already are features referencing third modules inside it (e.g. Orchard.Search.MediaLibrary, depending on Orchard.MediaLibrary).

**Review by sebastienros — APPROVED** [review]

**Pull request merged**